flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/ProducerConfigConstants.java

```diff
@@ -24,10 +24,28 @@
  */
 public class ProducerConfigConstants extends AWSConfigConstants {
 
+	/** Deprecated key. **/
+	public static final String DEPRECATED_COLLECTION_MAX_COUNT = "aws.producer.collectionMaxCount";
+
+ 	/** Deprecated key. **/
+	public static final String DEPRECATED_AGGREGATION_MAX_COUNT = "aws.producer.aggregationMaxCount";
+
 	/** Maximum number of items to pack into an PutRecords request. **/
-	public static final String COLLECTION_MAX_COUNT = "aws.producer.collectionMaxCount";
+	public static final String COLLECTION_MAX_COUNT = "CollectionMaxCount";
```

Comment:
Do we really want to expose these config keys now?
AFAIK, if the user wants to tweak these, with this change they should just simply refer to the KPL docs to see what keys are available. Explicitly exposing partially some keys in Flink but not others is a bit weird, IMO.